main.py

```diff
@@ -103,6 +108,8 @@ def __call__ (self, parser, namespace, values, option_string = None):
     parser.add_argument('--angle_front', type=float, default=60, help="[0, angle_front] is the front region, [180, 180+angle_front] the back region, otherwise the side region.")
     parser.add_argument('--t_range', type=float, nargs='*', default=[0.02, 0.98], help="stable diffusion time steps range")
 
+    parser.add_argument('--test_freq', type=int, default=None, help="Test model every n iterations")
```

Comment:
I'd clarify it, e.g. replace "Test model every n iterations" with "test the model every n iteration, by recording a turntable video. (By default, this is only done after the final epoch)"